### ISSUE TITLE
MAM Broken - Object reference not set to an instance of an object

### ISSUE THREAD

**Issue by kars85** [Jackett/Jackett#4159]
Traversing site works fine logged in through my normal workstation - went as far as going into security settings on MAM and creating a mam_id to put into Jackett, which must be a new change in the past week. Testing in jackett works fine after that, until searching for a book in lazylibrarian.

Both jackett and lazylibrarian are updated to latest git versions.

Jackett is at v0.10.431.0
Mono v5.16.0.179

Jackett is running from the linuxserver.io image, as is lazylibrarian.

-------------------------------
```

11-13 17:51:49 Error Jackett.Common.IndexerException: Exception (myanonamouse): Object reference not set to an instance of an object ---> System.Exception: Parse error ---> System.NullReferenceException: Object reference not set to an instance of an object


  at Jackett.Common.Indexers.BaseWebIndexer.OnParseError (System.String results, System.Exception ex) [0x00064] in <7bf52d42fcc7416894c852b62121fb70>:0 


  at Jackett.Server.Controllers.ResultsController.Torznab (Jackett.Common.Models.DTO.TorznabRequest request) [0x00335] in <a95ffd4cd56e4c6d911bfce5fa401800>:0  Jackett.Common.IndexerException: Exception (myanonamouse): Object reference not set to an instance of an object ---> System.Exception: Parse error ---> System.NullReferenceException: Object reference not set to an instance of an object


  at Jackett.Common.Indexers.BaseWebIndexer.OnParseError (System.String results, System.Exception ex) [0x00064] in <7bf52d42fcc7416894c852b62121fb70>:0 


11-13 17:51:49 Error Jackett.Common.IndexerException: Exception (myanonamouse): Object reference not set to an instance of an object ---> System.Exception: Parse error ---> System.NullReferenceException: Object reference not set to an instance of an object


  at Jackett.Common.Indexers.BaseWebIndexer.OnParseError (System.String results, System.Exception ex) [0x00064] in <7bf52d42fcc7416894c852b62121fb70>:0 


  at Jackett.Server.Controllers.ResultsController.Torznab (Jackett.Common.Models.DTO.TorznabRequest request) [0x00335] in <a95ffd4cd56e4c6d911bfce5fa401800>:0  Jackett.Common.IndexerException: Exception (myanonamouse): Object reference not set to an instance of an object ---> System.Exception: Parse error ---> System.NullReferenceException: Object reference not set to an instance of an object


  at Jackett.Common.Indexers.BaseWebIndexer.OnParseError (System.String results, System.Exception ex) [0x00064] in <7bf52d42fcc7416894c852b62121fb70>:0 
```

**Comment by kaso17** [comment]
please share the search URL used by LL

**Comment by kars85** [comment]
Hi @kaso17 - here's the debug logs within LL that include the URL pointing to my jacket docker container. Does it help if I paste the logs in table format, or with the code tags?
```
2018-11-14 04:42:52 | INFO | SEARCHBOOK | providers.py | BlockProvider | 393 | Blocking  provider  http://192.168.1.49:9117/api/v2.0/indexers/myanonamouse/results/torznab/  for 60 minutes because Jackett.Common.IndexerException: Excepti...

2018-11-14 04:42:52 | ERROR | SEARCHBOOK | providers.py | NewzNabPlus | 993 | http://192.168.1.49:9117/api/v2.0/indexers/myanonamouse/results/torznab  - Jackett.Common.IndexerException: Exception (myanonamouse): Object  reference not set to an instance of an object ---> System.Exception:  Parse error ---> System.NullReferenceException: Object reference not  set to an instance of an object   at Jackett.Common.Indexers.Myanonamouse.PerformQuery  (Jackett.Common.Models.TorznabQuery query) [0x0021a] in :0     --- End of inner exception stack trace ---   at Jackett.Common.Indexers.BaseWebIndexer.OnParseError (System.String results, System.Exception ex) [0x00064] in :0    at Jackett.Common.Indexers.Myanonamouse.PerformQuery (Jackett.Common.Models.TorznabQuery query) [0x004fd] in :0    at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery (Jackett.Common.Models.TorznabQuery query) [0x00091] in :0     --- End of inner exception stack trace ---   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery (Jackett.Common.Models.TorznabQuery query) [0x000ee] in :0    at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery (Jackett.Common.Models.TorznabQuery query) [0x0006b] in :0     at Jackett.Server.Controllers.ResultsController.Torznab  (Jackett.Common.Models.DTO.TorznabRequest request) [0x00335] in  <1855ca8cab084366a39764634f5bdbce>:0

2018-11-14 04:42:52 | DEBUG | SEARCHBOOK | providers.py | NewzNabPlus | 989 | Parsing results from http://192.168.1.49:9117/api/v2.0/indexers/myanonamouse/results/torznab
```

**Comment by kaso17** [comment]
ok, it was happening if there are no search results.
Fixed via 81035a002a857ad194dfef64a293aec0a86ac62e
